pkg/runtime/wfengine/wfengine.go

```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
 }
 
 var (
+	WorkflowActorType   = actors.InternalActorTypePrefix + "wfengine.workflow"
+	ActivityActorType   = actors.InternalActorTypePrefix + "wfengine.activity"
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID string
+}
+
+func getNamespace() string {
+	namespace := os.Getenv("NAMESPACE")
+	if namespace == "" {
+		namespace = defaultNamespace
+	}
+	return namespace
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	WorkflowActorType = WorkflowActorType + "." + config.AppID + "." + getNamespace()
+	ActivityActorType = ActivityActorType + "." + config.AppID + "." + getNamespace()
```

Comment:
nit: namespace should come before the app ID.

Reply:
Done